script/jenkins/check-changelogs.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+docker run --rm \
```

Comment:
I'd provision the docker container in the Jenkinsfile to not force to have docker to run the thing

Reply:
Not sure though, I'd rather prefer the script to be simple and then able to run it locally

Reply:
I've created https://github.com/elastic/apm-server/issues/2442